app/views/historic_appointments/_role_appointment.html.erb

```diff
@@ -1,11 +1,21 @@
 <% political_membership = role_appointment.historical_account.try(:political_membership) %>
-<%= content_tag_for :div, role_appointment, class: 'govuk-grid-column-one-quarter', role: "listitem" do %>
-  <%= render "govuk_publishing_components/components/image_card", {
-    href: historic_appointment_path(role.historic_param, role_appointment.person),
-    image_src: role_appointment.person.image_url(:s216),
-    image_loading: "lazy",
-    heading_text: role_appointment.person.name,
-    extra_links: previous_dates_in_office_prime(role, role_appointment.person, political_membership),
-    extra_links_no_indent: true
-  } %>
+  <%= content_tag_for :div, role_appointment, class: 'govuk-grid-column-one-quarter', role: "listitem" do %>
+    <% if historic_appointment_path(role.historic_param, role_appointment.person) != "/government/history/past-prime-ministers/theresa-may" %> 
```

Comment:
on the off chance that this problem hangs around for a long time, please add a comment to the code explaining why we've had to do this.

Reply:
Will do, I'll raise an issue as well linking it

Reply:
REF: https://github.com/alphagov/whitehall/issues/6266